notes/stories/film/film-scifi-2000s.st.txt

```diff
@@ -2515,6 +2515,59 @@ mother and son [Pluto Nash's mother Flura informed Pluto that Rex Crater had ass
 celebrity worship [The small time smuggler Felix Laranga was a big fan of Pluto Nash. He had Pluto sign his helmet, and blew Pluto's cover at the hotel/casino by informing everyone within earshot that Pluto was in the room.]
 
 
+movie: Simone (2002)
+====================
+
+:: Title
+Simone
+
+:: Description
+Simone (stylized as S1m0ne) is a 2002 American science fiction film written,
+produced, and directed by Andrew Niccol. It stars Al Pacino, Catherine Keener,
+Evan Rachel Wood, Rachel Roberts, Jay Mohr, and Winona Ryder.
+
+Synopsis: A fading director creates a virtual actress to star in his films and
+attempts to keep her non-human nature a secret as she becomes more and more
+famous.
+
+:: Date
+2002-08-23
+
+:: References
+https://en.wikipedia.org/wiki/Simone_(2002_film)
+
+:: Ratings
+1 <paul>
+
+:: Major Themes
+the desire for prestige [The story follows the fading film director Viktor Taransky as he takes the desperate measure of secretly inserting a virtual actress into his films in an effort to get the recognition he feels that he deserves. In Taransky's own words, "I wanted recognition" he said in regard to his motives for perpetrating this hoax.]
+coping with personal failure [The fading film director Viktor Taransky resorted to an elaborate hoax in an effort to change his fortunes after having suffered ten years of "abject failure in the film industry".]
+filmmaking [Viktor Taransky's using of a virtual actress in his films was a main novelty of the story.]
+old flames [The divorcees Viktor and Elaine rekindled their relationship over the course of the film, and in the end they got back together.]
+father and daughter [Viktor was advised by his precocious young daughter Lainey about this career and relationships over the course of the film, and he tried as he might to show he cared for her.]
+criminal fraud [Viktor Taransky clandestine use of a virtual actress in his hit films was tantamount to fraud.]
```

Comment:
Was it? Justify!

Reply:
Knocked down to minor with new motivation.